src/search/landmarks/landmark_cost_assignment.cc

```diff
@@ -19,18 +19,17 @@ using namespace std;
 namespace landmarks {
 LandmarkCostAssignment::LandmarkCostAssignment(
     const vector<int> &operator_costs, const LandmarkGraph &graph)
-    : empty(), lm_graph(graph), operator_costs(operator_costs) {
+    : lm_graph(graph), operator_costs(operator_costs) {
 }
 
 const set<int> &LandmarkCostAssignment::get_achievers(
```

Comment:
I would find the opposite parameter order more logical. But feel free to leave the way it is.